$NetBSD: patch-ab,v 1.4 2010/08/12 12:22:44 gdt Exp $

2008-12-20:
Don't use -O6 and -O3 because some gcc versions choke.
From Bernd Ernesti in private mail.
2010-08-12:
It is not clear if this is still true. -gdt

--- configure.orig	2009-07-26 22:36:34.000000000 +0200
+++ configure	2009-08-09 17:13:18.000000000 +0200
@@ -18101,7 +18101,7 @@
 
   else
 
-  for stp_ac_arg in -O6 -O3 -O2 -O1 -O ; do
+  for stp_ac_arg in -O2 -O1 -O ; do
     stp_ac_save_CFLAGS="${CFLAGS}"
 
   { $as_echo "$as_me:$LINENO: checking if ${CC} supports ${stp_ac_arg}" >&5
